modules/prebidServerBidAdapter/index.js

```diff
@@ -713,8 +713,8 @@ const OPEN_RTB_PROTOCOL = {
       return;
     }
     const request = {
-      id: s2sBidRequest.tid,
-      source: {tid: s2sBidRequest.tid},
+      id: firstBidRequest.auctionId,
+      source: {tid: firstBidRequest.auctionId},
```

Comment:
It is possible that publishers can pass in their own `auctionId`'s into `requestBids`

And in this case, we would want to be sure source.tid is always a UUID.

So after speaking with @bretg I think we should just generate a random UUID for source.tid here every time.

`utils.generateUUID()`

@bretg Can you comment and confirm?

Reply:
Good catch on the pub-specified auctionId. source.tid needs to be the same even if there are multiple PBS vendors. i.e. generate a new UUID for source.tid before we start looping through the vendors.

Reply:
Hi there, can we move forward on this PR? I can help if needed.
cc @idettman

Reply:
@osazos - your help would be appreciated.

Reply:
Hi @bretg, I finally got time to work on this one.

Regarding your last comment I would like to be sure we want this behavior because it is a breaking change. Actually a large part of the process is based on the fact the `tid` is unique by Vendor. See there: https://github.com/prebid/Prebid.js/blob/07ab54d60b4fa94fca3f922382eee508cfe7d629/src/adapterManager.js#L346-L370